AlertaDengue/locale/en/LC_MESSAGES/django.po

```diff
@@ -20,52 +20,22 @@ msgstr ""
 "X-Source-Language: pt_BR\n"
 "X-Generator: Poedit 1.8.7.1\n"
 
-#: AlertaDengue/common/templates/dbf_upload.html:66
-#: AlertaDengue/dbf/templates/dbf_upload.html:66
-msgid "Envie aqui os dados atualizados do SINAN"
-msgstr "Send here the updated SINAN data"
-
-#: AlertaDengue/common/templates/dbf_upload.html:172
-#: AlertaDengue/dbf/templates/dbf_upload.html:172
-msgid "Enviar"
-msgstr "Send"
-
-#: AlertaDengue/common/templates/dbf_upload.html:179
-#: AlertaDengue/common/templates/upload_successful.html:10
-#: AlertaDengue/dbf/templates/dbf_upload.html:179
-#: AlertaDengue/dbf/templates/upload_successful.html:10
-msgid "Ultimos arquivos enviados:"
-msgstr "Last uploaded files:"
-
-#: AlertaDengue/common/templates/upload_successful.html:5
-#: AlertaDengue/dbf/templates/upload_successful.html:5
-msgid "Dados do SINAN recebidos"
-msgstr "SINAN data received"
-
 #: AlertaDengue/dados/charts/cities.py:74
-#, fuzzy
-#| msgid "Limiar epidêmico"
 msgid "limiar pré epidêmico"
 msgstr "Epidemic threshold"
 
 #: AlertaDengue/dados/charts/cities.py:75
-#, fuzzy
-#| msgid "Limiar epidêmico"
 msgid "limiar pós epidêmico"
 msgstr "Epidemic threshold"
```

Comment:
should be "Post-epidemic threshold"

Reply:
Here, we don't need to translate list items for iteration (lines 74,75,76).

``` 
        ks_limiar = [
            _('limiar pré epidêmico'),
            _('limiar pós epidêmico'),
            _('limiar epidêmico'),
        ]

        colors = ['rgb(0,255,0)', 'rgb(255,150,0)', 'rgb(255,0,0)']

        for k, c in zip(ks_limiar, colors):
            figure.add_trace(
                go.Scatter(
                    x=df['SE'],
                    y=df[k],
                    y0=df['incidência'],
                    name=k.title(),
                    marker={'color': c},
                    text=df.SE.map(lambda v: '{}'.format(str(v)[-2:])),
                    hoverinfo='text',
                    hovertext=df['incidência'],
                    hovertemplate="Semana %{text} <br>Incidência=%{y:1f}",
                ),
                secondary_y=False,
            )
```